src/client-services/github.client-service.ts

```diff
@@ -1,8 +1,8 @@
-import axios, { AxiosResponse, AxiosInstance } from 'axios';
-import getConfig from 'next/config'
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import getConfig from 'next/config';
 
 function axiosConfig(): AxiosInstance {
-  const { serverRuntimeConfig } = getConfig()
+  const { serverRuntimeConfig } = getConfig();
```

Comment:
none blocker: not super familar with nextjs. but if this is server side function. we would want to ensure during server initialization the `serverRuntimeConfig.githubApiToken` must be defined before this method gets called. thus the service won't start until the right configurations are provided.

Reply:
serverRuntimeConfig is just a name we defined. it should probably be changed.

With that said, since we are running this in Github pages, it's all client side rendered.